omniscidb/DataProvider/TableFragmentsInfo.h

```diff
@@ -82,7 +80,6 @@ class FragmentInfo {
   mutable size_t numTuples;
   mutable ChunkMetadataMap chunkMetadataMap;
   mutable bool synthesizedNumTuplesIsValid;
```

Comment:
We might be able to remove this -- the invalidate calls were used for update/delete, which is all gone.

Reply:
You are right, it's not used anymore.

Reply:
No, I was wrong. It is removed in later patches when we don't store result sets there anymore.